### PR TITLE
Fixed setchangeInEvent bug

### PR DESCRIPTION
**Pull request by CristobalMe**
## Description
Fixed To Do No.10 in the Front end
https://docs.google.com/document/d/14icyb4vCgCw3JRqNwXO_NHtpBfacNtElH2HOWzSuoag/edit

### CODE REVIEW

File: frontend/src/components/eventPage/EventPage.jsx [hunk 1]
```diff
@@ -9,30 +9,30 @@ const EventPage = () => {
...
-        if (changeInEvent) {
-            setIsLoading(true)
-            fetchEvent()
-            fetchComments()
-        }
-        setIsLoading(false)
-        setchangeInEvent(false)
+        fetchEvent()
+        fetchComments()
     }, [event])
```
Comment (toriqyanM) [inline frontend/src/components/eventPage/EventPage.jsx:17]: https://www.w3schools.com/react/react_useeffect.asp
With your current logic, event and comments are re-fetched every time event state has changed. Is that what you want to achieve? Or do you want to re-fetch them every render?

File: frontend/src/components/homePage/HomePage.jsx [hunk 2]
```diff
@@ -27,13 +26,9 @@ function App() {
...
     useEffect(() => {
-        if (changeInEvents) {
-            setIsLoading(true)
-            fetchEvents()
-            fetchEventsWithCategory(currentCategory)
-        }
+        fetchEvents()
+        fetchEventsWithCategory(currentCategory)
         setIsLoading(false)
-        setChangeInEvents(false)
```
Comment (toriqyanM) [inline frontend/src/components/homePage/HomePage.jsx:36]: Same as above. When do you actually want to re-fetch events from db? Every time events state changed? But what does your fetchEvents change? Isn't it eventState?

File: frontend/src/components/eventPage/EventPage.jsx [hunk 3]
```diff
@@ -9,30 +9,30 @@ const EventPage = () => {
...
 
     const fetchEvent = () => {
         fetch(`http://localhost:3000${current_link}`)
             .then((response) => response.json())
-            .then((data) => setEvent(data))
+            .then((data) => {
+                setEvent(data)
+                setIsLoading(false)
```
Comment (toriqyanM) [inline frontend/src/components/eventPage/EventPage.jsx:24]: Is IsLoading ever set to true except from the first render?

File: frontend/src/components/eventPage/EventPage.jsx (hunk 1 above)
Comment (CristobalMe) [reply frontend/src/components/eventPage/EventPage.jsx:17]: Every render.
Thank you, I didn't know you could do this.

File: frontend/src/components/eventPage/EventPage.jsx (hunk 3 above)
Comment (CristobalMe) [reply frontend/src/components/eventPage/EventPage.jsx:24]: No

File: frontend/src/components/eventPage/EventPage.jsx (hunk 1 above)
Comment (CristobalMe) [reply frontend/src/components/eventPage/EventPage.jsx:17]: Would it be better to fetch it only in the first render?

**Pull request merged**